drivers/gles3/rasterizer_storage_gles3.cpp

```diff
@@ -152,6 +152,13 @@ void RasterizerStorageGLES3::GLWrapper::reset() {
 	texture_unit_table.blank();
 }
 
+int32_t RasterizerStorageGLES3::safe_gl_get_integer(unsigned int p_gl_param_name, int32_t p_max_accepted) {
```

Comment:
Seems odd to use glGetInteger64v but then take an int32 as the max accepted.

Reply:
There is no need for `int64_t` as the maximum as we do not use more than the 32 bit range in 3.x. This means that the calling code can store the results as 32 bit, which ensures existing code still works as is.